addons/mail/static/tests/qunit_suite_tests/components/message_tests.js

```diff
@@ -697,6 +697,46 @@ QUnit.test('prevent attachment delete on non-authored message in channels', asyn
     );
 });
 
+QUnit.test('allow attachment image download on message', async function (assert) {
+    assert.expect(2);
+
+    const pyEnv = await startServer();
+    const mailChannelId1 = pyEnv['mail.channel'].create({});
+    const irAttachmentId1 = pyEnv['ir.attachment'].create({
+        mimetype: 'text/plain',
+        name: "Blah.jpg",
+        mimetype: 'image/jpeg',
+    });
+    pyEnv['mail.message'].create({
+        attachment_ids: [irAttachmentId1],
+        body: '<p>Test</p>',
+        model: 'mail.channel',
+        res_id: mailChannelId1,
+    });
+    const { createThreadViewComponent, messaging } = await start();
+    const thread = messaging.models['Thread'].findFromIdentifyingData({
+        id: mailChannelId1,
+        model: 'mail.channel',
+    });
+    const threadViewer = messaging.models['ThreadViewer'].create({
+        hasThreadView: true,
+        qunitTest: insertAndReplace(),
+        thread: replace(thread),
+    });
+    await createThreadViewComponent(threadViewer.threadView);
```

Comment:
Prefer using a main component, such as `Discuss` with `openDiscuss` helper returned by `start()`.

We are currently working in changing tests so that they no longer mount small components like that.
Reasoning is that these tests are fragile, because it mounts a components without an actual and legit context that makes sense. So it can work by chance / thanks to implementation details, but would easily fail with small unrelated changes.

Reply:
@alexkuhn Changes Done!